src/swsssdk/configdb.py

```diff
@@ -170,37 +220,70 @@ def set_entry(self, table, key, data):
                   Pass {} as data will create an entry with no column if not already existed.
                   Pass None as data will delete the entry.
         """
-        key = self.serialize_key(key)
-        client = self.get_redis_client(self.db_name)
-        _hash = '{}{}{}'.format(table.upper(), self.TABLE_NAME_SEPARATOR, key)
-        if data == None:
-            client.delete(_hash)
+        if namespace is None:
+            namespace = self.ns_name
+
+        ns_instances = []
+        if self.mode == 'all':
+            ns_instances = self.get_redis_client_instances()
         else:
-            original = self.get_entry(table, key)
-            client.hmset(_hash, self.__typed_to_raw(data))
-            for k in [ k for k in original.keys() if k not in data.keys() ]:
-                if type(original[k]) == list:
-                    k = k + '@'
-                client.hdel(_hash, self.serialize_key(k))
-
-    def mod_entry(self, table, key, data):
+            ns_instances.append(namespace)
+
+        for instance in ns_instances:
+            key = self.serialize_key(key, instance)
+            _hash = '{}{}{}'.format(table.upper(), ConfigDBConnector.separators[instance], key)
+            client = self.get_redis_client(self.db_name, instance)
+            if client == None:
+                continue;
+
+            if data == None:
+                client.delete(_hash)
+            else:
+                original = self.get_entry(table, key)
+                client.hmset(_hash, self.__typed_to_raw(data))
+                for k in [ k for k in original.keys() if k not in data.keys() ]:
+                    if type(original[k]) == list:
+                        k = k + '@'
+                    client.hdel(_hash, self.serialize_key(k, instance))
+
+    def mod_entry(self, table, key, data, create_new_flag=False, namespace=None):
```

Comment:
>mod_entry [](start = 8, length = 9)

I see reason to add `namespace` to `connect()`, but no reason to add each other function.